CMakeLists.txt

```diff
@@ -464,6 +464,13 @@ if (NOT EMSCRIPTEN)
     )
   endif ()
 
+  # Python 3.5 is the version shipped in Ubuntu Xenial
+  find_package(PythonInterp 3.5)
+  if(NOT PYTHONINTERP_FOUND)
```

Comment:
how about AND BUILD_TESTS == ON there?

Otherwise lgtm!

Reply:
added this check.